Create nginx.md

ankytastic opened the pull request:
## This PR fixes...

_This PR adds the Dictionary definition of "nginx"_

## What I did...

_Add dictionary definition of "nginx"._

## How to test...

_In a list format, tell us how to test this PR._
Ex:

1. Navigate to /Dictionary
2.Search "nginx"
3.See the definition and perspective show up under "nginx"
4.Check grammar and typos

## I learned...

_Learnt it is called as "engine-x"_

ankytastic committed:
Create nginx.md

ankytastic commented:
[@cherryOnTechAdmin ](https://github.com/cherryontech) please accept my pr and label it for hacktoberfest

alliequintano commented on src/assets/content/dictionary/nginx.md:
```diff
@@ -0,0 +1,10 @@
+---
+title: nginx
+definition: nginx [engine x] is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server, originally written by Igor Sysoev.
```
```suggestion
definition: Nginx [engine x] is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server, originally written by Igor Sysoev.
```

alliequintano reviewed (changes requested):
Hi @ankytastic, thank you for contributing to our tech dictionary! Your definition looks good, i've just made a tiny capitalization fix. 

For the perspective, what we are really looking for is to consider a role on a software team such as QA, developer, designer, etc., and write a perspective from that role. I find the example in the [CONTRIBUTING.md ](https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md) helpful to clarify this. Also keep in mind how the value will be evaluated in the sentence, which you can see on the site either testing locally or via the Netlify deploy preview. You can find additional instructions in the CONTRIBUTING.md on that as well.

<img width="1731" alt="image" src="https://user-images.githubusercontent.com/3057984/194982135-c5ec5f16-8ec4-42b7-8602-3152e33d320f.png">

ankytastic committed:
Update src/assets/content/dictionary/nginx.md

Co-authored-by: Allie Quintano <alliequintano@gmail.com>

aviku352 committed:
Update SPA.md

ankytastic committed:
Merge pull request #1 from aviku352/pit

Update SPA.md

alliequintano commented on src/assets/content/dictionary/SPA.md:
```diff
@@ -4,7 +4,7 @@ definition: SPA stands for Single Page Application. It is a very common way of p
 - sourceurl: https://betterprogramming.pub/how-to-implement-a-single-page-application-using-react-router-cc6b9e5c3aec#:~:text=SPA%20stands%20for%20Single%20Page,content%20without%20refreshing%20the%20website.
 perspectives:
-- meaning: SPA is used to make the user experience feel a lot smoother. You can tell when it’s a SPA or multi-page application when navigating between menus often because a multi-page application will reload, making the whole UI blink fast depending on the content.
+- meaning: a way for me to make the user experience feel a lot smoother. You can tell when it’s a SPA or multi-page application when navigating between menus often because a multi-page application will reload, making the whole UI blink fast depending on the content
```
Did you mean to introduce these fixes with this PR?

snyk-bot committed:
fix: package.json & yarn.lock to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-POSTCSS-1090595
- https://snyk.io/vuln/SNYK-JS-POSTCSS-1255640

ankytastic committed:
Merge pull request #2 from ankytastic/snyk-fix-64085a16dff5ebb5e11953bd6bd1b70b

[Snyk] Fix for 2 vulnerabilities
